tests/garage/experiment/test_snapshotter.py

```diff
@@ -19,41 +20,37 @@
 }), ('none', {})]
 
 
-class TestSnapshotter:
+class TestSnapshotter(TfGraphTestCase):
```

Comment:
Does this test involve using a tensorflow graph?

Reply:
Seems not. Updated.